spec/aria-reflection.html

```diff
@@ -133,52 +133,52 @@ <h2>Interface Mixin
       </h2>
       <pre class="idl">
     interface mixin AriaAttributes {
-    attribute DOMString? ariaActiveDescendant;
-    attribute DOMString? ariaAtomic;
-    attribute DOMString? ariaAutoComplete;
-    attribute DOMString? ariaBusy;
-    attribute DOMString? ariaChecked;
-    attribute DOMString? ariaColCount;
-    attribute DOMString? ariaColIndex;
-    attribute DOMString? ariaColSpan;
-    attribute DOMString? ariaControls;
-    attribute DOMString? ariaCurrent;
-    attribute DOMString? ariaDescribedBy;
-    attribute DOMString? ariaDetails;
-    attribute DOMString? ariaDisabled;
-    attribute DOMString? ariaErrorMessage;
-    attribute DOMString? ariaExpanded;
-    attribute DOMString? ariaFlowTo;
-    attribute DOMString? ariaHasPopup;
-    attribute DOMString? ariaHidden;
-    attribute DOMString? ariaInvalid;
-    attribute DOMString? ariaKeyShortcuts;
-    attribute DOMString? ariaLabel;
-    attribute DOMString? ariaLabelledBy;
-    attribute DOMString? ariaLevel;
-    attribute DOMString? ariaLive;
-    attribute DOMString? ariaModal;
-    attribute DOMString? ariaMultiLine;
-    attribute DOMString? ariaMultiSelectable;
-    attribute DOMString? ariaOrientation;
-    attribute DOMString? ariaOwns;
-    attribute DOMString? ariaPlaceholder;
-    attribute DOMString? ariaPosInSet;
-    attribute DOMString? ariaPressed;
-    attribute DOMString? ariaReadOnly;
-    attribute DOMString? ariaRelevant;
-    attribute DOMString? ariaRequired;
-    attribute DOMString? ariaRoleDescription;
-    attribute DOMString? ariaRowCount;
-    attribute DOMString? ariaRowIndex;
-    attribute DOMString? ariaRowSpan;
-    attribute DOMString? ariaSelected;
-    attribute DOMString? ariaSetSize;
-    attribute DOMString? ariaSort;
-    attribute DOMString? ariaValueMax;
-    attribute DOMString? ariaValueMin;
-    attribute DOMString? ariaValueNow;
-    attribute DOMString? ariaValueText;
+    attribute DOMString ariaActiveDescendant;
+    attribute DOMString ariaAtomic;
```

Comment:
It seems like this *could* be a boolean if we could figure out the
right reflection semantics.

Basically any boolean where the default is false is plausible, right?

Reply:
I think in order to make that work we'd need to define new IDL types. Possibly something something similar to what I drafted in this [comment on ARIA Issue #691](https://github.com/w3c/aria/issues/691#issuecomment-367891305).